package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A discord bot created for use in a personal discord server.",
   "main": "index.js",
   "scripts": {
-    "dev": "nodemon src/app.ts"
+    "dev": "nodemon -r dotenv/config src/app.ts",
```

Comment:
We spoke about this, you said you were testing nodemon + dotenv and it didnt work. So remove this please!